src/api.js

```diff
@@ -24,5 +24,14 @@ const getArticles = () => {
       });
   };
 
+  const getCommentsByArticleId = (id) => {
+    return api.get(`/articles/${id}/comments`).then ((response)=>{
+      return response.data.comments
+    })
+    .catch((error)=>{
```

Comment:
I would probably not worry about handling the error hear, it already throws so just handle it where and when you need to